### PR TITLE
Upgrade Kotlin, AGP and the Gradle wrapper

### PR DESCRIPTION
**Pull request by vRallev**

### CODE REVIEW

File: lib/build.gradle [hunk 1]
```diff
@@ -14,7 +14,6 @@ android {
 dependencies {
   api deps.recyclerview
 
-  implementation deps.kotlin_stdlib
```
Comment (vRallev) [inline lib/build.gradle:17]: I remove the dependency because the Kotlin Gradle Plugin adds it by default.

File: lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt [hunk 2]
```diff
@@ -306,7 +307,8 @@ private class MutationExtension<T : Any>(val spec: MutationExtensionSpec<T>) : E
 
   override lateinit var data: RecyclerData<T>
 
-  internal fun setUpDragHandle(view: View) {
+  @SuppressLint("ClickableViewAccessibility")
+  fun setUpDragHandle(view: View) {
```
Comment (zach-klippenstein) [inline lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt:311]: Why make this public? Doesn't seem related.

**Review by zach-klippenstein — APPROVED** [review]

File: lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt (hunk 2 above)
Comment (vRallev) [reply lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt:311]: It's redundant, because the class itself is private.
